### DISCUSSION TITLE
Ошибка веб-приложения в телеграмм при скролле

### DISCUSSION THREAD

**Discussion by 1Mantih1** [flet-dev/flet#3393] [Q&A]
Здравствуйте! Используя flet на python, столкнулся с проблемой сворачивания веб-приложения в телеграмм при скролле вертикально. Как это исправить? В интернете нашёл множество решений для js, но для flet - ничего

**Comment by ndonkoHenri** [comment]
English please.

**Comment by 1Mantih1** [comment]
Hello!  Using flet in python, I encountered a problem with the telegram web application collapsing when scrolling vertically.  How to fix it?  I found a lot of solutions for js on the Internet, but nothing for flet.

**Comment by ndonkoHenri** [comment]
It will be nice if you could provide more details as required by the bug template. Code to reproduce, image or video showing the issue etc

**Reply by MoltenMind** [reply]
Got the same issue on IOS. On Android the situation is even worse - the scrolling is very twitchy. I think the reason lies in the constant display of the scroll bar (despite the fact, that scroll mode is set to AUTO), you can see it on the video below.
Android:

https://github.com/user-attachments/assets/ba39a7ec-96f4-4c2a-855a-0d618a4549eb

IOS:

https://github.com/user-attachments/assets/e51b5ed4-44ad-407d-a540-9b38d01c43ee

The code is like this:

    import flet as ft
    def main(page: ft.Page):
        page.padding = ft.padding.symmetric(horizontal=page.width * 0.025, vertical=page.height * 0.03)
        page.update()
        page.clean()
        bags_grid = ft.GridView(
            expand=True,
            runs_count=2,
            max_extent=320,
            child_aspect_ratio=0.65,
            spacing=15,
            run_spacing=15,
        )
    
        def create_product_tile():
            return ft.Container(
                content=ft.Column(
                    controls=[
                        ft.Image(src="thumbs.dreamstime.com/z/red-leather-women-bag-isolated-over-white-24385120.jpg",
                                 fit="cover", height=150, width=150, border_radius=20),
                        ft.Text("Сумка", weight="bold", size=14),
                        ft.Text(f"Цена: 10000₽"),
                        ft.ElevatedButton(
                            content=ft.Row(
                                controls=[
                                    ft.Icon(ft.Icons.ADD_SHOPPING_CART_ROUNDED, color=ft.Colors.BLACK),
                                    ft.Text("Добавить в корзину", color=ft.Colors.BLACK, size=11),
                                ],
                                alignment="center",
                                spacing=1,
                            ),
                            width=250,
                            bgcolor=ft.Colors.WHITE,
                            style=ft.ButtonStyle(
                                shape=ft.RoundedRectangleBorder(radius=8)
                            ),
                        ),
                    ],
                    horizontal_alignment="center",
                    spacing=1,
                ),
                border_radius=ft.border_radius.all(20),
                padding=8,
                alignment=ft.alignment.center,
                bgcolor=ft.Colors.WHITE,
                width=100,
                ink=True,
            )
    
        for i in range(0, 4):
            bags_grid.controls.append(create_product_tile())
    
        info_row = ft.Row(controls=[ft.ElevatedButton(
            text="МЕНЕДЖЕР ДЛЯ ЗАКАЗА",
            width=200,
            color="black",
            bgcolor="white")],
            alignment="center", spacing=20, )
    
        main_page = ft.Column(
            controls=[
                info_row,
                ft.ElevatedButton(content=ft.Row(controls=[ft.Icon(name=ft.Icons.FILTER_LIST, color=ft.Colors.BLACK),
                                                           ft.Text("Нажмите для выбора фильтров", color=ft.Colors.BLACK)],
                                                 alignment="center"),
                                  bgcolor=ft.Colors.WHITE),
                bags_grid,
            ],
            horizontal_alignment="center",
            spacing=10,
            scroll=ft.ScrollMode.AUTO,
            height=page.height*0.94
        )
        page.add(main_page)
        page.update()
    ft.app(main, view=ft.AppView.WEB_BROWSER)

**Comment by lrrrtm** [comment]
Использую aiogram3, веб-приложение на flet открывается без проблем и не сворачивается, проверьте свой код flet.

**Reply by sn3g0v1k** [reply]
когда листаешь снизу вверх закрывается webapp
